linq/UnitTest1.cs

```diff
@@ -69,24 +69,17 @@ public static void v()
                 var bottom = startingDeck.Skip(26);
     
                 var shuffle2 = top.InterleaveSequenceWith(bottom);
-                foreach (var c in shuffle2)
-                {
-                    Console.WriteLine(c);
-                }
     
     
                 var times = 0;
                 var shuffle = startingDeck;
                 do
                 {
-                    shuffle = shuffle.Skip(26).InterleaveSequenceWith(shuffle.Take(26)).ToArray();
-                    /*
-    
-                    foreach (var c in shuffle)
-                        Console.WriteLine(c);
-    
-                    Console.WriteLine();
-                    */
+                    var result = shuffle.Skip(26).InterleaveSequenceWith(shuffle.Take(26)).ToArray();
+                    if( times % 2 == 0 )
+                        Assert.Equal( shuffle[times/2 + result.Length/2], result[times] );
+                    else Assert.Equal( shuffle[times/2], result[times] );
+                    shuffle = result;

```

Comment:
Not what I meant, it **has to** be in it's own loop, you should **not** use `times` variable in the assert - just like the console.writeline had it's own loop every single time, this has to have it's own loop as well, to pretty much replace it...